src/pages/experimental/import-pattern/index.mdx

```diff
@@ -48,21 +53,99 @@ Wherever possible there should be a drag and drop zone. Use a file drop componen
 <Row>
 <Column colLg={8}>
 
-![Example of loaded files in the import modal](/images/4.png)
+![Example of loaded files in the import modal](/images/03.png)
+
+<Caption>Example of a loaded file in the import modal</Caption>
+
+</Column>
+</Row>
+
+### Import from a URL (modal)
+
+If your product supports importing from a URL, use this method.
+
+<Row>
+<Column colLg={8}>
+
+![Example of importing with a URL](images/04.png)
+
+<Caption>Example of importing with a URL</Caption>
+
+</Column>
+</Row>
+<Row>
+<Column colLg={8}>
+
+![Example of a file imported by URL](images/05.png)
+
+<Caption>Example of a file imported by URL</Caption>
+
+</Column>
+</Row>
+
+## Choose a file (full page or side-panel)
+
+Use the drag and drop file upload component and offer the option to browse locally to select a file. Note that file selection should be restricted to allowed file types.
+
+If additional actions (e.g. adding metadata) are required to be performed on the file(s) they should be done separately to the file upload, prior to the import action being completed.  This could be in a subsequent step or if there is sufficient space include those actions in a single step.
```

Comment:
```suggestion
If additional actions (e.g. adding metadata) are required to be performed on the file(s) they should be done separately to the file upload, prior to the import action being completed. This could be in a subsequent step or if there is sufficient space include those actions in a single step.
```